### PR TITLE
fix(medusa, medusa-payment-paypal): Add missing data in Payment Collection input

### PR DESCRIPTION
**Pull request by adrien2p**
**What**

Payment collection should provider the region_id, total and id in the partial cart data

FIXES CORE-991

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]
LGTM

File: .changeset/nine-crabs-dress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/medusa": patch
```
Comment (olivermrbl) [inline .changeset/nine-crabs-dress.md:2]: todo: Should bump Klarna + PayPal

**Review by olivermrbl — APPROVED** [review]
LGTM, please update changeset + tests before merging :)

**Comment by adrien2p** [comment]
> LGTM, please update changeset + tests before merging :)

Yes will add some tests, I was caught in a call sorry :D

**Pull request merged**